src/python/director/lcmUtils.py

```diff
@@ -11,6 +11,8 @@
 import sys
 import re
 
+import drake
```

Comment:
Hello from Big Bear Lake!

Don't add dependency to this file.  There's another lcm message util file that has DRC message dependency, add there.